src/frontend/components/LiveVideoInformationBar/LiveControlButtons/index.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Box } from 'grommet';
+import { normalizeColor } from 'grommet/utils';
+
+import { theme } from 'utils/theme/theme';
+
+import { StudentShareDocButton } from './StudentShareDocButton';
+import { StudentShowViewersButton } from './StudentShowViewersButton';
+import { StudentShowChatButton } from './StudentShowChatButton';
+import { StudentJoinDiscussionButton } from './StudentJoinDiscussionButton';
+import { StudentQuitDiscussionButton } from './StudentQuitDiscussionButton';
+
+const LiveButtonsWrapper = styled(Box)`
+  div {
+    position: relative;
+
+    span {
+      position: absolute;
+      bottom: -2px;
+      right: 0px;
+      background-color: ${normalizeColor('status-warning', theme)};
+      border-radius: 8px;
+      color: ${normalizeColor('white', theme)};
+      font-size: 0.6em;
+      padding: 0 8px;
+      border: 2px solid ${normalizeColor('white', theme)};
+    }
+  }
+`;
+
+interface LiveControlButtonsProps {
+  parent: string;
+}
+
+export const LiveControlButtons = ({ parent }: LiveControlButtonsProps) => {
+  return (
+    <LiveButtonsWrapper direction="row" align="center" responsive={true}>
+      <StudentShareDocButton />
+
+      <StudentShowViewersButton />
+
+      <StudentShowChatButton />
+
+      {parent === 'LiveJitsi' ? (
```

Comment:
I don't know the idea with this `parent` but this is not the way we have for now to check if a student joined the discussion or not. You should use the `useParticipantWorkflow`. If `accepted` is `true` then the student joined the discussion.